go.mod

```diff
@@ -4,27 +4,29 @@ go 1.13
 
 require (
 	github.com/MakeNowJust/heredoc v0.0.0-20170808103936-bb23615498cd
-	github.com/containers/buildah v1.14.10
-	github.com/containers/common v0.8.4
-	github.com/containers/image/v5 v5.4.3
-	github.com/containers/storage v1.18.2
+	github.com/containers/buildah v1.15.1-0.20200716185836-11db2c1af990
+	github.com/containers/common v0.15.2
+	github.com/containers/image/v5 v5.5.1
+	github.com/containers/storage v1.20.2
```

Comment:
Not necessarily a _bad_ idea to bump buildah for a strict proof of concept. Personally I would have just run
```
go get github.com/containers/storage@v1.20.2
go mod tidy
go mod vendor
```
and left the `go mod` changes in place.

Reply:
That was my first try, @adambkaplan . However, it needs a bump in `buildah` as well, driven by the following error:
```
vendor/github.com/containers/buildah/run_linux.go:142:72: cannot use d.DeviceRule.Permissions (type configs.DevicePermissions) as type string in argument to g.AddLinuxResourcesDevice
make: *** [Makefile:11: build] Error 2
```

Reply:
That's from github.com/opencontainers/runtime-tools.  I'd strongly prefer we stay on the buildah release-1.15 branch if we can (11db2c1af990 is from the master branch), until we have a 1.16.

Reply:
@adambkaplan going back on this commit, I was able to do exactly what you mention. Please consider latest changes.